src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sumTotal = 0;
+  const costs = 40;
+  const discountOne = 20;
+  const discountTwo = 50;
+
+  if (days >= 7) {
+    sumTotal = days * costs - discountTwo;
+  } else if (days >= 3) {
+    sumTotal = days * costs - discountOne;
+  } else {
+    sumTotal = days * costs;
+  }
+
+  return sumTotal;
```

Comment:
you can return the value in each `if` block
```suggestion
  if (days >= 7) {
    return days * costs - discountTwo;
  }
  if (days >= 3) {
    return days * costs - discountOne;
  }
  return days * costs;
```